lib/theme/text_theme.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'colors.dart';
+
+/// The referenced design (Figma) is based on the Unity Style Typography. But
+/// the one we are supposed to follow is Material Design Typography. Each [TextStyle]
+/// defined below corresponds to the matching text style defined in Unity.
```

Comment:
Good that you found what the difference between a document comment vs a simple comment is (`///` vs `//` respectively). Here though you are not documenting your code for other callers to read. You are documenting weird behaviour. I suggest you use a simple comment instead.